specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -37,7 +37,9 @@ directive:
       - $.definitions.ManagedResource
   - suppress: TopLevelResourcesListBySubscription
     reason: The list by scope includes also list by subscription, this is an extension resource.
-    from: AlertsManagement.json    
+    from: AlertsManagement.json
+  - suppress: OperationsAPIImplementation
+    reason: The API is implemented in Operation API - false positive
 ```
```

Comment:
I dont follow the reasoning . Why do you not have the operations api defined in the swagger ? 

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/operations-api-implementation.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/operations-api-implementation.md)